realis.go

```diff
@@ -940,6 +942,10 @@ func (r *realisClient) RollbackJobUpdate(key aurora.JobUpdateKey, message string
 	return resp, nil
 }
 
+/* Admin functions */
+// TODO(rdelvalle): Consider moving these functions to another interface. It would be a backwards incompatible change,
```

Comment:
we can have version number to distinguish this kind of breaking change, it seems we haven't bump version number for a while ... https://github.com/paypal/gorealis/releases

Reply:
Agreed, I want to pile up a bunch of breaking changes into a V2. I should keep a list of this somewhere in the repo.

Reply:
We can add a CHANGELOG.md in root of repo and keep those stuff under unreleased section, so people can know we have new features and use them if they want to force version using commit hash . i.e. https://github.com/gaocegege/maintainer/blob/master/CHANGELOG.md